packages/ui/react-ui-editor/src/components/Toolbar/Toolbar.tsx

```diff
@@ -137,6 +140,91 @@ const ToolbarButton = ({ Icon, children, ...props }: ToolbarButtonProps) => {
   );
 };
 
+//
+// View Mode
+//
+
+const ViewModes: { [key: string]: { label: string; icon: Icon } } = {
+  'live-preview': { label: 'live preview mode label', icon: PencilSimple },
```

Comment:
'preview', 'readonly', 'source'
also:
`ViewModes = Record<ViewModeDefFromBelow, { label: string, icon: Icon }>`

(able to use def as key using Record, but not a plain `{}` map).